cfme/test_framework/appliance.py

```diff
@@ -44,11 +44,11 @@ def appliances_from_cli(cli_appliances):
 def pytest_configure(config):
 
     reporter = terminalreporter.reporter()
-    if stack.top:
-        appliances = [stack.top]
-    elif config.getoption('--dummy-appliance'):
+    if config.getoption('--dummy-appliance'):
```

Comment:
looks good at first glance, we still might want to know why stack.top is set here, i suspect a deeper issue and would love to see  the root cause in future